src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -713,6 +733,25 @@ public BitbucketTeam getTeam() throws IOException, InterruptedException {
         return getRepositories(null);
     }
 
+    /**
+     * Returns the account of the current owner or {@code null} if the current owner is not an account user.
+     *
+     * @return the account profile of the current owner, or {@code null} if current owner is not an account user.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    @CheckForNull
+    private BitbucketCloudUser getUser() throws IOException, InterruptedException {
```

Comment:
Is this the current user logged in? Somewhat :confused: by the API endpoint name

Reply:
Yes, it would be the user logged in by the creds. I'll rename it to `getUserInfo` and update the javadoc to clarify

Reply:
Thanks for clarifying and also clarify in code :sheep: